hosted_control_planes/hcp-manage/hcp-manage-virt.adoc

```diff
@@ -50,3 +50,5 @@ include::modules/hcp-virt-image-caching.adoc[leveloffset=+2]
 * xref:../../virt/virtual_machines/creating_vms_custom/virt-creating-vms-by-cloning-pvcs.adoc#smart-cloning_virt-creating-vms-by-cloning-pvcs[Cloning a data volume using smart-cloning]
 
 include::modules/hcp-virt-etcd-storage.adoc[leveloffset=+2]
+include::modules/hcp-virt-attach-nvidia-gpus.adoc[leveloffset=+2]
```

Comment:
Should these 2 new topics be set at level 2 or level 1? I'm not sure whether they should be nested under the "Configuring storage..." section.

Reply:
Right. I think I can shift them to  the level 1.